services/ingestion/etl/2014tc16rfcb047/xls/src/lib/transform.js

```diff
@@ -103,33 +101,60 @@ const getProjectId = record =>
         .update(record['Operation/Project Name'])
         .digest('hex');
 
+/**
+ * Gets country code from a country name.
+ *
+ * @memberof 2014tc16rfcb047XlsTransform
+ * @param {String} countryName The name of the country
+ * @returns {String} The ISO 3166-1 country code
+ */
+
+const getCodeByCountry = countryName =>
+  countries.getAlpha2Code(countryName, 'en');
+
 /**
  * Preprocess `project_locations`.
  *
  * Input fields taken from the `record` are:
  *
- * - `Operation Postcode`
  * - `Country`
  *
- * @memberof 2014tc16rfcb014CsvTransform
+ * @memberof 2014tc16rfcb047XlsTransform
  * @param {Object} record The row received from parsed file
  * @returns {Array<Location>}
  */
 
 const getLocations = record => {
+  // Tiny utility, as .flat() is not available in node.
+  const flatten = arr =>
+    arr.reduce((flat, toFlatten) => {
+      return flat.concat(
+        Array.isArray(toFlatten) ? flatten(toFlatten) : toFlatten
+      );
+    }, []);
+
   const locations = [];
-  const postCode = record['Operation Postcode'];
-  const country = record.Country;
-
-  locations.push({
-    address: country,
-    centroid: null,
-    country_code: '',
-    location: null,
-    nuts: [],
-    postal_code: postCode,
-    region: '',
-    town: '',
+  const countriesMulti = record.Country.split(',')
```

Comment:
what about just `const countriesMulti = record.Country.split(/\s*(?:,|&)\s*/);`? 😃 

it splits when it finds `,` or `&`, with the surrounding spaces (if any) (equivalent to trim)

the only difference is that it doesn't trim the beginning or the end of the input string

I guess `const countriesMulti = record.Country.trim().split(/\s*(?:,|&)\s*/);` would be a more exact equivalent...